MANIFEST.in

```diff
@@ -1,2 +1,5 @@
 recursive-include shoop *.jinja *.html *.txt
 recursive-include shoop_tests *.jinja *.html *.txt
+prune shoop/admin/static_src/*/bower_components
+prune shoop/admin/modules/*/bower_components
+prune shoop/*/node_modules
```

Comment:
Those are source files that will be compiled into bundles in `shoop/admin/static` etc -- running `npm run build` should be done before `sdist`ing.